Sources/StreamChatUI/ChatChannelList/ChatChannelAvatarView_Tests.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelAvatarView_Tests: XCTestCase {
+    var currentUserId: UserId!
+    var channel: ChatChannel!
+    
+    override func setUp() {
+        super.setUp()
+        currentUserId = .unique
+
+        channel = ChatChannel.mockDMChannel(members: [
+            .mock(id: currentUserId, imageURL: TestImages.vader.url),
+            .mock(id: .unique, imageURL: TestImages.yoda.url, isOnline: true)
+        ])
+    }
+    
+    func test_emptyAppearance() {
+        let view = ChatChannelAvatarView().withoutAutoresizingMaskConstraints
+        view.addSizeConstraints()
+        AssertSnapshot(view, variants: [.small])
+    }
+
+    func test_defaultAppearance_withDirectMessageChannel() {
```

Comment:
on `ChannelListVC_Tests` we have the pattern `test_channelListVC_defaultAppearance`. We should have the same pattern in all places. Although the latter is a bit repetitive, I think it generates a better snapshot name, especially when thinking of storing the artefacts on Github, it will be easier to implement, and it will be easier to understand what view the snapshot belongs to. I'm fine with both strategies, we just should make sure that we have the same pattern always 🙂 I guess if we have this strategy, we can just store the failure snapshots in a folder with the name of the view, so the decision will be just a question of taste 👍

Reply:
Hmm, I don't think we need to use the name of the view we test in the test name, since it's already in the call name. The snapshot are organized by the test case name in folders and it seems to me it's clear enough. WDYT?

<img width="1006" alt="Screen Shot 2021-02-11 at 12 35 27" src="https://user-images.githubusercontent.com/6388510/107631648-a0c9ab80-6c65-11eb-8c10-fcee8363b4b2.png">

Reply:
Yes for me its good like it is 👍